hummingbot/connector/gateway/gateway_price_shim.py

```diff
@@ -38,17 +38,17 @@ class GatewayPriceDeltaEntry:
 
 class GatewayPriceShim:
     """
-    Developer / QA tool for modifying the apparent prices on DEX connectors (e.g. uniswap) during integration tests.
+    Developer / QA tool for modifying the apparent prices on DEX connectors (e.g. amm) during integration tests.
 
-    When the gateway price shim is enabled for a particular DEX trading pair (e.g. "WETH-DAI" on uniswap), the apparent
+    When the gateway price shim is enabled for a particular DEX trading pair (e.g. "WETH-DAI" on amm), the apparent
```

Comment:
We have connector / exchange named uniswap
https://hummingbot.org/exchanges/uniswap/

this changes here not needed